src/Microsoft.AspNetCore.Mvc.Core/MvcOptions.cs

```diff
@@ -35,6 +35,18 @@ public MvcOptions()
         }
 
         /// <summary>
+        /// Gets or sets the flag which decides whether input formatters (for example, using
+        /// <see cref="FromBodyAttribute"/>) should allow no input to be provided.
+        /// <see langword="false"/> by default.
+        /// </summary>
+        /// <example>
+        /// When <see langword="false"/>, actions that model bind the request body (for example,
+        /// using <see cref="FromBodyAttribute"/>) will register a validation error if the incoming
+        /// request body is empty.
```

Comment:
The net result is that `ModelState.IsValid` becomes `false`. Don't developers think of that as a "validation error"? It's called `IsValid`, after all.

Have rephrased to "register an error in the ModelStateDictionary".